src/main/java/cuchaz/enigma/gui/elements/MenuBar.java

```diff
@@ -156,22 +156,22 @@ public MenuBar(Gui gui) {
 			}
 			menu.addSeparator();
 			{
-				JMenuItem stats = new JMenuItem("Mapping Stats...");
+				JMenuItem stats = new JMenuItem(LangUtils.translate("menu.file.stats"));
 
 				stats.addActionListener(event -> {
-                    JFrame frame = new JFrame("Choose Included Members");
+                    JFrame frame = new JFrame(LangUtils.translate("menu.file.stats.title"));
 					Container pane = frame.getContentPane();
 					pane.setLayout(new FlowLayout());
 
 					Map<StatsMember, JCheckBox> checkboxes = Arrays
 							.stream(StatsMember.values())
 							.collect(Collectors.toMap(m -> m, m -> {
-								JCheckBox checkbox = new JCheckBox(Utils.caplisiseCamelCase(m.name()));
+								JCheckBox checkbox = new JCheckBox(LangUtils.translate("type." + m.name().toLowerCase()));
```

Comment:
For all these `toLowerCase`, use `toLowerCase(Locale.ROOT)` so we aren't borked by turkish i again